sdk/tables/Azure.Data.Tables/samples/Sample4QueryEntities.md

```diff
@@ -96,3 +96,31 @@ foreach (Page<TableEntity> page in queryResultsMaxPerPage.AsPages())
     }
 }
 ```
+
+In more advanced scenarios, it may be necessary to store the continuation token returned from the service so your code controls exactly when the next pages is fetched.
```

Comment:
Is the idea that you would cache off the continuation token and get the next set of results later in your app?

Reply:
Yup